### ISSUE TITLE
Seeing "Rotki failed to start" message at the bottom of every page and left sidebar gets cut off when scrolling on certain pages

### ISSUE THREAD

**Issue by BigPapa0** [rotki/rotki#728]
failed to start: https://imgur.com/a/sZPDsNo

sidebar cut off: https://imgur.com/a/dcU58KE

sidebar completely cut off and can't navigate back (I clicked on the Statistics tab): https://imgur.com/a/PCXvmjh

[rotki_electron.log](https://github.com/rotki/rotki/files/4210853/rotki_electron.log)

**Comment by LefterisJP** [comment]
Hey @BigPapa0 thanks for the issue. This is a continuation of another problem also reported in https://github.com/rotki/rotki/issues/726

Important to note here that Rotki works but somehow you get these UI problems after logging in as far as I can undestand. Right?

@kelsos Can you help here? What could this be? The user is in Windows 10 and Rotki v1.1.1.

**Comment by BigPapa0** [comment]
Working fine except the premium key is not saved and I had to enter it again

**Comment by kelsos** [comment]
@LefterisJP at windows we also have the #709 and in the logs I can
see 
```
2020-02-16T19:55:31.706Z: The Python sub-process exited with signal: null (Code: 4294967295)
```
That most probably means that the sub-process failed to start due to a previously unterminated process.

As for the UI I will have to check. It is definitely weird looking.

**Comment by LefterisJP** [comment]
@BigPapa0 is this solved for you in Rotki v1.2.0?

**Comment by LefterisJP** [comment]
After 21 days of no response I will assume it's solved and close the issue. If that is not the case please re-open or if you see any other issue please open a new issue.
